.gitignore

```diff
@@ -3,3 +3,6 @@ node_modules
 .nuxt
 nuxt.d.ts
 .DS_Store
+.output
```

Comment:
No need to remove this from ignore, but what is it?

Reply:
Static site generation with Nuxt 3 outputs the static site files to the `.output/public` subdirectory. So, this is equivalent to ignoring the `dist` subdirectory in Nuxt 2.